app/main/forms.py

```diff
@@ -1392,35 +1392,44 @@ def __init__(self, *args, **kwargs):
     )
 
 
-class BrandingOptionsEmail(StripWhitespaceForm):
+class BrandingOptions(StripWhitespaceForm):
 
     FALLBACK_OPTION_VALUE = 'something_else'
     FALLBACK_OPTION = (FALLBACK_OPTION_VALUE, 'Something else')
 
-    options = RadioField('Choose your new email branding')
+    options = RadioField('Choose your new branding')
     something_else = TextAreaField('Describe the branding you want')
 
-    def __init__(self, service, *args, **kwargs):
+    def __init__(self, service, *args, branding_type="email", **kwargs):
         super().__init__(*args, **kwargs)
-        self.options.choices = tuple(self.get_available_choices(service))
+        self.options.choices = tuple(self.get_available_choices(service, branding_type))
+        self.options.label.text = 'Choose your new {} branding'.format(branding_type)
         if self.something_else_is_only_option:
             self.options.data = self.FALLBACK_OPTION_VALUE
 
     @staticmethod
-    def get_available_choices(service):
+    def get_available_choices(service, branding_type):
+        if branding_type == "email":
+            organisation_branding_id = service.organisation.email_branding_id if service.organisation else None
+            service_branding_id = service.email_branding_id
+            service_branding_name = service.email_branding_name
+        elif branding_type == "letter":
+            organisation_branding_id = service.organisation.letter_branding_id if service.organisation else None
+            service_branding_id = service.letter_branding_id
+            service_branding_name = service.letter_branding_name
 
         if (
             service.organisation_type == Organisation.TYPE_CENTRAL and
-            service.organisation.email_branding_id is None and
-            service.email_branding_id is not None
+            organisation_branding_id is None and
+            service_branding_id is not None
         ):
             yield ('govuk', 'GOV.UK')
 
         if (
             service.organisation_type == Organisation.TYPE_CENTRAL and
             service.organisation and
-            service.organisation.email_branding_id is None and
-            service.email_branding_name.lower() != 'GOV.UK and {}'.format(service.organisation.name).lower()
+            organisation_branding_id is None and
+            service_branding_name.lower() != 'GOV.UK and {}'.format(service.organisation.name).lower()
```

Comment:
GOV.UK and organisation should only be available as email branding
```suggestion
            service_branding_name.lower() != 'GOV.UK and {}'.format(service.organisation.name).lower() and 
            branding_type == "email"
```